neo_force_scheme/_neo_force_scheme.py

```diff
@@ -200,11 +203,19 @@ def transform(
         if random_state is not None:
             np.random.seed(random_state)
 
-        # randomly initialize the projection
         if starting_projection_mode is not None:
+            # randomly initialize the projection
             if starting_projection_mode == ProjectionMode.RANDOM:
+                #print(starting_projection_mode)
                 Xd = np.random.random((size, 2))
-
+            # initialize the projection with tsne
+            else:
+                if starting_projection_mode == ProjectionMode.TSNE:
```

Comment:
its better to use elif instead of else: (new line) if:

Reply:
there are other places with the same thing happening, so better to fix all of them